Lab 1/Lab1.html

```diff
@@ -0,0 +1,286 @@
YXaew7FfhAatNZeScUaFY9ix7j+4tG1C7Vh0athjAjI0neSCA/TOXt18GUzkUeYd9Hw5nwIAF7Lz4EHtXoVzO7WHtJ8Jh7dpB0EPwyJAPxFwuYynA6dN85TqlCiFPhEKy5NTKmYwa+REbrtnj7f4MZ/MzbJk3lYHvbeDsY1eeFehILEWanjhZcPWXpYxp14z6tWpRp7EhvQQIOm/lLB8uLrIDkk7CgXWy4taeLwVdRjBp6xUcnzliY3qePWs+YVJ3I2WNOwxhmjBIF1SD5ORoxZ1jO1kxawRdOhoJ2rWn4+h5rPnjNrbOSg60k5s917bOZWgnQ9p26Erv3j3p0bEthq2n8MXPwiD5eeF695AAzZMxnvcT1x2e8cyxiLs56wyR3WW2fDKWASMXsvGCFR5uNpic+41vZxkzwEjITtuOdB84k1V7LmEh1WR2dsD20W1O/PgF00b2kvVNu07dhW7dxF+3hZGXo8s2WFzex/rP3qNHh/YyzYx6TGXprvNYOCupGFKJ0UdHvmFYr+l8deB2AUNflMOaa7vnMWjAeBbtE+8l1sjO5AS/fP2hcFI7KfPtMIqZ3xySgaicymVnxrUjW1gybgA9JX3Ttgv9xixky0kTwSPSfIU8Od1h71fT6SLRtGs/hn28kT9vWAtHXNFHDipocnF8wNnNnzCirbA57bsw4CNhs+4JYP8WdYmDGp11un+UdR+OYMDkley/74K7cHxuHd3G0mkj6STr1U50G/oZG47cwVayIxIoldNwhNN/ey8r3utHW2E723SZynKhNy3FGjs52fHw+lF+XjGbAV06Kvaq4zg+2SLZK6XRlVQtRSpZ7CBkZs3UwXQ2bE3rXhP5fMc5zB0L6qu3U0XqyXMzjiwcy+ABH7Lh6APsnjtjefkQmxdNpl/3jrJObTdgIp9tP4OpnbPAEtJBeSeeCrD+6MJPLBrfW+iD1nQeJIDtwes8dlabflld5fdlM+gz6huOWhbo8i6fdbBVZOnz/LK0RJIlpydFg0XkIg0uPPcOJCTYH09J5mXw/pTnnr4EhkUQESGNUAJ9pH4QL+fk/NybkFB/vJ45q6Wm89phR7nso4uHPyERkUSGhwnQGUpoeASRkUH4eagdq50K6nYph92f4ABPOZhW5P1CpB1RH0GbEH+85TQYySY/x9MvSMxL0CUygvCwwJd9ZuT+ShI9/QkN9hZ2qvA5SGlwHj4BhEWo9A2XaKI8Q0lldOKJdA5G/E5YuHqPeAcvuRBG4btc/z5wL20FyiDFUQilN4Gh0kTDCNY1p1FLLikEDSA4yA8PXa6nxCgBYTJhwkNDCA4JI1wiUqgw0E5FbXgcZCD6zEu8c7C0aE7KgWBp0X0DhBCEKwsqGMdX2r7SLaiUkuDmhZ9uLhIt/DyU5gmqsDjYC0Xk5U+Q/AwhVIHeuLk4qUyhq08rlJybN76Cji6PrbBzdsdPau7kqJTzKjJF4+KmdEP18xQOnYMMul2ee+IbGJI739Ag75eNQ5wKPzzm4uYj6Oorz1s5e/FMKI2XdA0PC1AattjnKW9o78xzvwB83YRBs7TF1ccPX0+pa1wRgjepbJ0QcGku7mo01PmpolxCw1QeCVHm4via95Tya129/AgM9MVTRzdnV/EMwXsBHoVEFnRd9pxkHgkIcMfh0WNsXHzk+584FDGYdVQqfriIdwsODiJAMhTy1rdkkMQ8BV1kXpG6/nk/z/O+Yj6u7nj7BwuFqfKB0BeeajMdBzWa5eIpDJKkUKWujMEhgs6SEpcUuIsSaXryHJ/AAAJ9n8uHCYtDy3hZpqXKDb6BQgf6ykZaWj/5oKR/kDKfiHDCJAOh4xXZIAgQ4i7WODhMNTJCXwid4Oqsc/AlnST4TTw3LI/s+Li55Kv+IBn64FDF+Dg6OBQLmsh6wN1X8IhYu+cqeBJ87yZ0Y4hOdsKD8Zc6Q0rrKKfJCH3j6iHWN0Q1quFC3/gI2XPK5RF7AUrdJMOs8oikfyV62ecNNKjy4+LuR3CE8pwQ6T4nxZa9db365JmsM4IDvOUGZXL3VSELkq4NV98vTOgYqbyeQ56zI7J+dH6Ot3R+S6ZXfr0pnQlz885Lz6Bce6UAHeUZ9k7i9wMkHS6AnmTT3Fxeq6/+7x0eZ54JwBYcLGT/meSw6cBW4EudGqraXVV3yEFGac5PhH0KErSIjCQ8RAG2Mi1kkKwA05BATxmbOBRW7vofyFLR9MWwEXjL5mUKjBQwEH+zkdIb1WFjW6CXkLQzLL7zprNGDna2+Z6hDPU7Tq/bXRHvYlPUxQle6lU7lTa2Ki6SeyPY2BSYj32Bg8S24u+vTymS+i/Z2uZ/ho1t/mcU/juv163/wsi9Uz6C5TKKTcGmVYUwqPy3goylfLd4GJ8872z/+kW3zdc8pwAd5PnYvlIHNp9QSd3g7AsyhfocGyXPWk7hsLEt+lbPUp64zeM8wq17z/zzfXU+ryqVvMIl1zN+E111Ck0Ik41aFUW637Y4dBuV555/LoUqlzc8J69sOOVTom/qAKijvbQWDgpP2RSfTooO9raK0dEZl39gkF6hnc2rtHutQbJ3UHMm7ZVnFBs9kscQ5eEVp3/CKw6FyZddvp27f8RvOkNvX7yqkTnYKe+d+14qj+Q3qnYFdrvsFB30Wn2j09v5Py9cvxewWUVFH1m3Pc4j74WBB9tC9YhDPnrkX3sJ2NoVBlL+yTMcitLuPlbtrsNr7YNNIe/4Jlq85IvX69T/RneXjJLxTytA/WvAvY0QAmc5cl+ycP/3NX9L6FCwNnYJHQr0HiihQ8koGSWjZJSMklHshoSVJcxcbMG9RqOR/5Ve8NGjRzKosJNOUpeMklEySkbJKBklo2SUjJJRMvINCStLmDk4ODgfli52kfuoqCi5yYObmxvPnz8vGSWjZJSMklEySkbJKBklo2QUGBJWljBzZGRk8Yzcl1wlV8lVcpVcJVfJVXKVXCVXyVW01/9zcC95Ha8MjUbeatCNQu/5m+9oNNp/9r0
```

Comment:
This histogram is pretty squashed. I think you could have made the same point with less histograms or alternatively stretch out the x-axis to make it more clear.

Reply:
Also titles, legends, and axis labels would make all of your plots more understandable